### PR TITLE
Added more detail for ConfereceMode parameter to Update New-CsCallQueue.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/office-docs-powershell/issues/5702.

Thank you @CLYVR!

### CODE REVIEW

File: skype/skype-ps/skype/New-CsCallQueue.md [hunk 1]
```diff
@@ -296,7 +296,11 @@ Accept wildcard characters: False
 ```
 
 ### -ConferenceMode
-The ConferenceMode parameter indicates whether or not Conference mode will be applied on calls for this Call queue. Conference mode significantly reduces the amount of time it takes for a caller to be connected to an agent, after the agent accepts the call.
+The ConferenceMode parameter indicates whether or not Conference mode will be applied on calls for this Call queue. Conference mode significantly reduces the amount of time it takes for a caller to be connected to an agent, after the agent accepts the call. The following bullets detail the difference between both modes:
```
Comment (get-itips) [inline skype/skype-ps/skype/New-CsCallQueue.md:299]:
```suggestion
The ConferenceMode parameter indicates whether or not Conference mode will be applied on calls for this Call queue. Conference mode significantly reduces the amount of time it takes for a caller to be connected to an agent, after the agent accepts the call. The following bullet points detail the difference between both modes:
```

File: skype/skype-ps/skype/Set-CsCallQueue.md [hunk 2]
```diff
@@ -311,7 +311,11 @@ Accept wildcard characters: False
 ```
 
 ### -ConferenceMode
-The ConferenceMode parameter indicates whether or not Conference mode will be applied on calls for this Call queue. Conference mode significantly reduces the amount of time it takes for a caller to be connected to an agent, after the agent accepts the call.
+The ConferenceMode parameter indicates whether or not Conference mode will be applied on calls for this Call queue. Conference mode significantly reduces the amount of time it takes for a caller to be connected to an agent, after the agent accepts the call. The following bullets detail the difference between both modes:
```
Comment (get-itips) [inline skype/skype-ps/skype/Set-CsCallQueue.md:314]:
```suggestion
The ConferenceMode parameter indicates whether or not Conference mode will be applied on calls for this Call queue. Conference mode significantly reduces the amount of time it takes for a caller to be connected to an agent, after the agent accepts the call. The following bullet points detail the difference between both modes:
```

**Comment by dariomws** [comment]
Thanks, @CLYVR and @get-itips!

**Comment by yogkumgit** [comment]
@chrisda Please merge this PR. Thanks.

**Pull request merged**